images/update_kcp_stable/Dockerfile

```diff
@@ -0,0 +1,35 @@
+FROM registry.access.redhat.com/ubi8/ubi-minimal:8.6
```

Comment:
I would rename the image to `update_pipeline_service`. `kcp_stable` is the name of our repo.